PAWI_HS18_Projektdokumentation.tex

```diff
@@ -388,10 +388,15 @@ \section{Systemarchitektur}
 \newpage
 \section{Komponentendesign}
 
+Damit unsere Applikation funktioniert sind mehrere Teile nötig, in Abbildung \ref{fig:Komponentendesign} sind die wichtigsten schematisch dargestellt. Wir gebrauchen neben den dargestellten natürlich auch ein Betriebsystem (und dazugehörende Komponenten wie Speichermanagement und weiteres), aus Gründen der Übersichtlichkeit sind diese jedoch nicht aufgeführt.
```

Comment:
Achtung wieder sehr viel wir gebraucht mein Vorschlag:

Die Entwickelte Applikation ist von weiteren Kompoenenten und Teilsystemen abhängig. In der Folgenden Graphik sind die wichtigsten Komponenten, welche für die Funktionstüchtigkeit notwendig sind abgebildet. Aus Gründen der Übersichtlichkeit sind keine weiteren Komponenten aufgelistet.

Reply:
Kleiner Pet Peeve: Anstatt den Pull Request zu schliessen, beim Review Changes anfordern. Dann sieht man was in der Queue ist.

Reply:
Sorry 👍  wird in Zukunft so gelöst.